src/form/acr.jl

```diff
@@ -278,10 +278,10 @@ end
 
 ""
 function sol_data_model!(pm::AbstractACRModel, solution::Dict)
-    if haskey(solution, "nw")
-        nws_data = solution["nw"]
+    if haskey(solution["it"]["ep"], "nw")
+        nws_data = solution["it"]["ep"]["nw"]
     else
-        nws_data = Dict("0" => solution)
+        nws_data = Dict("0" => solution["it"]["ep"])
     end
 
```

Comment:
should use the `_pm_it_name` but this is also an opportunity to leverage one of IM's apply to all power networks helper functions.

Reply:
Makes sense, yes.